src/lib/apiErrorHandler.js

```diff
@@ -221,6 +221,10 @@ export const handleWorkerError = async (job, error, logContext) => {
  */
 export const handleUnexpectedWorkerError = (job, error, logContext) => {
   logger.error(`${logContext.namespace} - Unexpected error in catch block - ${error}`);
+  if (error instanceof Sequelize.ConnectionAcquireTimeoutError) {
+    logger.error(`${logContext.namespace} - Critical error: Restarting server.`);
```

Comment:
Please make the error message state that this is the Sequelize.ConnectionAcquireTimeoutError that is the critical error